### ISSUE TITLE
Create builders for common actions

### ISSUE THREAD

**Issue by Stratus51** [Stratus51/rust_dash7_alp#32]
Building actions is too cumbersome.

```
Action::ReadFileData(action::ReadFileData {
    file_id: 1,
    offset: 2,
    size: 4,
})
```

Example:
```
Action::read_file(fid=1, offset=2, size=4)
```

**Comment by Stratus51** [comment]
This should only be done for simple actions such as Nop, ReadFile, WriteFile.

Actions such as ActionQuery would take so many arguments that the builder call would be unreadable.
And I don't want to encourage that kind of code.

**Comment by Stratus51** [comment]
closed by #35
